components/calculation/PopulationComparisonTable.tsx

```diff
@@ -313,28 +273,54 @@ export default function PopulationComparisonTable({ patientId, defIds }: Populat
             })}
           </tbody>
         </Table>
-        <Autocomplete
-          data={Object.keys(defIds).sort((a, b) => (a < b ? -1 : 1))}
-          value={searchValue}
-          onChange={setSearchValue}
-          dropdownComponent={ScrollArea}
-          maxDropdownHeight={200}
-          placeholder="Expression Name"
-          icon={<Search />}
-          nothingFound={
-            <Text align="left" style={{ paddingLeft: 10 }}>
-              No Matches
-            </Text>
-          }
-          limit={100}
-          label="Search CQL Expression Definition"
-          onItemSubmit={item => {
-            document.getElementById(defIds[item.value])?.scrollIntoView({ behavior: 'smooth' });
-          }}
-        />
       </>
     );
   } else {
     return <Text>Measure highlighting not available</Text>;
   }
 }
+
+export function PopulationComparisonTableControl() {
```

Comment:
Convention is usually to not put 2 components in the same file especially if one doesn't render the other. Can you move `PopulationComparisonTableControl` to its own file?